client/components/planning-editor-standalone/storage-adapter.ts

```diff
@@ -15,7 +16,18 @@ export const storageAdapterPlanningItem: IStorageAdapter<IPlanningItem> = {
         const fieldDefinitions = getFieldDefinitions();
         const fieldStorageAdapter = fieldDefinitions[fieldId]?.storageAdapter;
 
-        if (fieldStorageAdapter != null) {
+        if (fieldId.includes(SUBJECT_PREFIX_ID)) {
```

Comment:
use `.startsWith` instead of `.includes`

Regarding `SUBJECT_PREFIX_ID` - would be good if we could avoid this approach of checking contents of an ID. I suggested the idea to assure you that it can be done using existing vocabulary fields from client-core. Since we agree now that it can be done, we can look into how to do it better :) It's a bit messy that you run storage adapter twice for vocabulary fields. Once in field definition and second time in main storage adapter. Change `IFieldDefinition['storageAdapter']` so you can do it all in once place.